install/ansible/istio/tasks/enable_mutating_webhooks.yml

```diff
@@ -24,16 +25,30 @@
       with_items:
         - webhook-create-signed-cert.sh
         - webhook-patch-ca-bundle.sh
-  when: cluster_flavour == 'ocp'
+  when:
+    - cluster_flavour == 'ocp'
+    - istio_version_to_use < minimum_istio_version_for_webhook_certificates
 
-- name: Create signed certificate
-  shell: "{{ istio_dir }}/install/kubernetes/webhook-create-signed-cert.sh --service istio-sidecar-injector --namespace istio-system --secret sidecar-injector-certs"
 
-- name: Install the sidecar injection configmap
-  shell: "{{ cmd_path }} apply -f {{ istio_dir }}/install/kubernetes/istio-sidecar-injector-configmap-release.yaml"
+- block:
+    - name: Create signed certificate
+      shell: "{{ istio_dir }}/install/kubernetes/webhook-create-signed-cert.sh --service istio-sidecar-injector --namespace istio-system --secret sidecar-injector-certs"
```

Comment:
These files will not be present in the release, so this conditional logic can be removed.  If you look in:
https://github.com/istio/istio/tree/master/install/kubernetes

You will see no files are present this conditional logic relies upon.  From my understanding, we don't backport features or improvements so there is no viable way this code would be usable as is.

The goal of 0.8 is to be very stable and serve as an LTS until 1.0 is released.  I'd like the Ansible implementation to be ready to roll for at least deployment of Istio on Kubernetes.